types.go

```diff
@@ -206,9 +206,6 @@ type Order struct {
 }
 
 type OrderBookEntry struct {
```

Comment:
Are we able to instead add `ID` before generating, rather than removing? I know this it he output of the generator, but technically this would constitute a breaking change

Reply:
I've asked Donovan to put it back in using the sdkgenerator but the problem is atm the way it has been done is neither consistent with the other languages or even really supported by the server.